.postcssrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
 			preserve: false,
 			features: {
 				'custom-properties': false,
+				'focus-within-pseudo-class': false,
```

Comment:
[Postcss focus within](https://github.com/csstools/postcss-focus-within) needs to be paired with the [focus-within polyfill](https://github.com/jsxtools/focus-within) in order to work. Otherwise it just swaps the selector for an attribute that cannot be targetted. Considering that [:focus-within](https://caniuse.com/css-focus-within) is supported by all browsers except for IE, it made more sense to just turn off the postcss feature but correct me if I'm wrong and we're supporting IE 11.